sdk/table/azure-table/azure/table/_message_encoding.py

```diff
@@ -0,0 +1,150 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+# pylint: disable=unused-argument
+
+from base64 import b64encode, b64decode
+
+import sys
+import six
+from azure.core.exceptions import DecodeError
+
+from ._shared.encryption import decrypt_queue_message, encrypt_queue_message
+
+
+class MessageEncodePolicy(object):
```

Comment:
If this is an abc class, make it abc and "encode" an abstract method
https://github.com/Azure/azure-sdk-for-python/blob/7e0ddbe546a07b9b77b4c6b7fb17f812a0c29467/sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/crypto/_internal/transform.py#L20